owtf/webapp/src/components/ConfigurationTabsNav/index.js

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
@sharmamohit123 Can you make a folder in components named, SettingsPage and put ConfigurationTabsNav.js in that. That way it will be more organized.